### ISSUE TITLE
Security vulnerability introduced in tutorial/mqtt MQTT Bridge

### ISSUE THREAD

**Issue by muhlpachr** [bigclownlabs/bc-doc#34]
MQTT traffic from MQTT broker in security zone Clown.Safe must not leave hub without extra authorization, otherwise MQTT messages can be compromised/faked easily.
Any traffic through Internet MUST NOT be clear (unsecured by TLS e.g. as proposed in tutorial example).
MQTT Bridge outside Clown.Safe zone should be strongly discouraged or there must be risks explanation and recommendations for safe configuration. 
Could be also confusing regarding AWS IoT.

**Comment by vlaxa** [comment]
We can secure communication by SSH tunnel like this: `ssh -f -L 22883:127.0.0.1:1883 pirate@IP_of_broker -N` and then client will connect locally to `127.0.0.1:22883`

**Comment by hubpav** [comment]
Yes, exactly. In the future we have agreed to secure MQTT via some channel, e.g. SSH or secure MQTT itself (which is unlikely to happen with current mechanisms). Right now the Mosquitto is unsecure, yes, however I feel we are already discussing this on multiple levels and in addition we are bringing it on top in issues with multicolor labels :-)

**Comment by muhlpachr** [comment]
@hubpav be aware there is proposed configuration to export naked MQTT to Internet zone (interconnect MQTT broker in zone Clown.Safe with MQTT broker in Internet zone)!!!
That should be strongly discouraged regardless of transport unless appropriate measures are implemented. 
Any integration between Clown.Safe and Internet zones has to designed with "Principle of least privilege" and appropriate measures (e.g. adaptive fw filters etc).

We have agreement for users and Clown.Safe zone mainly.
I am not aware of any agreement for appropriate measures for interconnection of various MQTT brokers and especially bring  MQTT traffic form Clown.Safe zone to MQTT broker in Internet zone.

IMHO there are not enough colors to WARN "Do not do it!", "Do not provide examples for it!".
